services/src/handlers/plots.rs

```diff
@@ -189,14 +206,87 @@ mod tests {
 
         assert_eq!(
             result,
-            json!([{
-                "pixel_count": 6,
-                "nan_count": 0,
-                "min": 1.0,
-                "max": 6.0,
-                "mean": 3.5,
-                "stddev": 1.707_825_127_659_933
-            }])
+            json!({
+                "plot_type": "Json",
+                "plot_name": "Statistics",
+                "data": [{
+                    "pixel_count": 6,
+                    "nan_count": 0,
+                    "min": 1.0,
+                    "max": 6.0,
+                    "mean": 3.5,
+                    "stddev": 1.707_825_127_659_933
+                }]
+            })
+            .to_string()
+        );
+    }
+
+    #[tokio::test]
+    async fn chart() {
```

Comment:
I think we have to be very careful with upgrading the vega library because we don't test how the plots actually look.

Reply:
We actually do since the definition is checked in `datatypes`.

Reply:
we only check how the vega string looks like but not how the rendered plot looks like.

Reply:
Yes, but we can't do this before we haven't figured out how to get a png server-wise.